lib/internal/worker.js

```diff
@@ -441,7 +441,7 @@ function setupChild(evalScript) {
 
   port.on('message', (message) => {
     if (message.type === messageTypes.LOAD_SCRIPT) {
-      const { filename, doEval, workerData, publicPort, hasStdin } = message;
+      let { filename, doEval, workerData, publicPort, hasStdin } = message;
```

Comment:
```suggestion
      const { filename, doEval, workerData, publicPort, hasStdin } = message;
```

Reply:
It's not ok to move all those identifiers to let, better we add new const, see below